src/visualizers/VisualizerHistogram.ts

```diff
@@ -0,0 +1,203 @@
+import {VisualizerExportModule} from '@/visualizers/VisualizerInterface'
+import {VisualizerDefault} from './VisualizerDefault'
+import {natlog} from 'c:/Users/devli/Desktop/Numberscope/frontscope/src/shared/math'
+
+class HistogramVisualizer extends VisualizerDefault {
+    name = 'Histogram'
+
+    binSize = 1
+    terms = 10
+    firstIndex = 1
+    linear = false
+
+    params = {
+        binSize: {
+            value: this.binSize,
+            forceType: 'integer',
+            displayName: 'Bin Size',
+            required: true,
+        },
+
+        firstIndex: {
+            value: this.firstIndex,
+            forceType: 'integer',
+            displayName: 'First Index',
+            required: true,
+        },
+
+        terms: {
+            value: this.terms,
+            forceType: 'integer',
+            displayName: 'How many terms of the series',
+            required: true,
+        },
+    }
+
+    checkParameters() {
+        const status = super.checkParameters()
+
+        if (this.params.binSize.value < 1) {
+            status.isValid = false
+            status.errors.push('Bin Size can not be less than 1')
+        }
+
+        if (this.params.binSize.value < 1) {
+            status.isValid = false
+            status.errors.push('Bin Size can not be less than 1')
+        }
+
+        if (this.params.firstIndex.value < 1) {
+            status.isValid = false
+            status.errors.push('First index can not be less than 1')
+        }
+
+        return status
+    }
+
+    //finding the largest value of the sequence asked for
```

Comment:
Can you change the comments so they are in the [imperative mood](https://en.wikipedia.org/wiki/Imperative_mood)?